### DISCUSSION TITLE
Larger chameleons

### DISCUSSION THREAD

**Discussion by Batman2741** [eccentricdevotion/TARDIS#630] [Ideas]
### Describe the feature request

Just simply discussing the Tardis as buildings larger than the current 3x3

### Describe alternatives you've considered

Just adding Clara's dinner exterior

**Comment by eccentricdevotion** [comment]
Another one that has been discussed and rejected in the past...

**Reply by jpg39** [reply]
Is it not possible to make this configurable, or would that be a total rewrite of the plug-in?

**Comment by jpg39** [comment]
Completely agree — we’ve seen the TARDIS exterior as large as an ancient pyramid (Logopolis) or a house (the Master), and as small as a shoebox.  It would be super cool to disguise the TARDIS as an actual village house when landing in one, or an actual pyramid when landing in the desert.

**Reply by Batman2741** [reply]
Exactly

**Comment by Batman2741** [comment]
![dd9kucb-ba04c078-1c8c-46f5-b19a-7345eca31afc.png](https://user-images.githubusercontent.com/104590478/168931442-36fb07f1-5c90-4feb-8db0-a6c2a07ad912.png)
